ui/platform_window/x11/x11_window_base.cc

```diff
@@ -97,7 +97,9 @@ X11WindowBase::X11WindowBase(PlatformWindowDelegate* delegate,
 }
 
 X11WindowBase::~X11WindowBase() {
+  delegate_->OnAcceleratedWidgetDestroying();
   UnConfineCursor();
+  delegate_->OnAcceleratedWidgetDestroyed();
```

Comment:
place it after destroy? Wdyt? Can you check what's the best destruction order here?